### PR TITLE
Fix LoremIpsum import for esm 

### PR DESCRIPTION
**Pull request by hath995**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fast-check",
-  "version": "1.7.0",
+  "version": "1.7.1",
```
Comment (dubzzz) [inline package.json]: Please do not bump manually, I will on my side with a specific commit.

File: src/check/arbitrary/LoremArbitrary.ts [hunk 2]
```diff
@@ -1,18 +1,25 @@
 // tslint:disable-next-line:no-var-requires
```
Comment (dubzzz) [inline src/check/arbitrary/LoremArbitrary.ts]: I think we can remove this line now

**Comment by hath995** [comment]
I think if we added a headless browser test to make sure it can run in the browser.

https://docs.travis-ci.com/user/chrome
https://developers.google.com/web/updates/2017/04/headless-chrome

I may give this a shot this weekend.

**Comment by dubzzz** [comment]
I will publish the new version tomorrow morning (Paris time)

**Pull request merged**